Too many return values error on loadController()

Hi, Firstly this project looks great. Really looking forward to working with it.

Getting the error...

bud/.app/controller/controller.go:133:15: too many return values
        have (nil, error)

Looks the loadController() is missing a return error definination

`func loadController() *controller.Controller {
	controllerController, err := controller.Load()
	if err != nil {
		return nil, err
	}
	return controllerController
}`

--- a/framework/controller/controller_test.go
+++ b/framework/controller/controller_test.go
@@ -1940,3 +1940,34 @@ func TestStructInStruct(t *testing.T) {
 	is.Equal(result.Stdout(), "")
 	is.Equal(result.Stderr(), "")
 }
+
+// https://github.com/livebud/bud/issues/101
+func TestLoadController(t *testing.T) {
+	is := is.New(t)
+	ctx := context.Background()
+	dir := t.TempDir()
+	td := testdir.New(dir)
+	td.Files["controller/controller.go"] = `
+		package controller
+		func Load() (*Controller, error) {
+			return &Controller{}, nil
+		}
+		type Controller struct {}
+		func (c *Controller) Index() string { return "" }
+	`
+	is.NoErr(td.Write(ctx))
+	cli := testcli.New(dir)
+	app, err := cli.Start(ctx, "run")
+	is.NoErr(err)
+	defer app.Close()
+	// Test GET /
+	res, err := app.Get("/")
+	is.NoErr(err)
+	res.Diff(`
+		HTTP/1.1 200 OK
+		Content-Type: application/json
+
+		""
+	`)
+	is.NoErr(app.Close())
+}

--- a/framework/controller/loader.go
+++ b/framework/controller/loader.go
@@ -515,6 +515,7 @@ func (l *loader) loadContext(controller *Controller, method *parser.Function) *C
 				Import: importPath,
 				Type:   recv.Type().String(),
 			},
+			&di.Error{},
 		},
 		Params: []di.Dependency{
 			di.ToType("net/http", "ResponseWriter"),

